libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageViewModel.kt

```diff
@@ -35,14 +42,24 @@ class PreviewImageViewModel : ViewModel() {
             currentPosition = currentPosition,
             loadSuccess = true
         )
-        // TODO: handle file save on done button action to be added later
-//        val highResImageJustLoadedIntoView = newState != currentState && newState == ImageInHighResLoadSuccessUiState
-//        val imageNotLoadedIntoFile = loadIntoFile.value !is ImageLoadToFileSuccessState
-//        if (highResImageJustLoadedIntoView && imageNotLoadedIntoFile) {
-//            updateLoadIntoFileState(ImageStartLoadingToFileState(currentUrl))
-//        }
-        val currentUiState = uiState.value as UiState
-        updateUiState(currentUiState.copy(viewPagerItemsStates = newImageUiStates))
+        val newImageState = newImageUiStates[currentPosition]
```

Comment:
`currentPosition` seems a bit misleading name for the parameter. It's not the position of the currently selected item but a position of any item which has just finished loading, right? I'd consider renaming it to `position`, wdyt?

Reply:
Renamed here: 2867882